libs/base/ui/action/icon-button/src/icon-button.styles.ts

```diff
@@ -2,11 +2,12 @@ import { css } from 'lit';
 
 export const iconButtonStyles = css`
   :host {
+    --oryx-icon-color: var(--oryx-color-neutral-11);
```

Comment:
No, this is not what I meant. You do not need to make any changes in this component, if you need to change it's color just set this css var in the styles where this is necessary, like there:

`oryx-icon-button {
    --oryx-icon-color: red;
}`